opensearch/src/main/java/org/opensearch/sql/opensearch/executor/OpenSearchExecutionEngine.java

```diff
@@ -49,7 +51,14 @@ public void execute(PhysicalPlan physicalPlan, ExecutionContext context,
               result.add(plan.next());
             }
 
-            QueryResponse response = new QueryResponse(physicalPlan.schema(), result);
+            String rawResponse;
+            try {
+              rawResponse = ((OpenSearchIndexScan) ((ProjectOperator) physicalPlan).getInput())
+                      .getRawResponse();
+            } catch (Exception e) {
```

Comment:
Is is possible to be more specific that `Exception` here? This will catch everything and silence it. 
Even better if we could avoid the try...catch block if possible (and check to see if the `RawResponse` exists...

Reply:
Changed in https://github.com/Bit-Quill/opensearch-project-sql/commit/c7208ea1f35ee9d657fb6898e800edb77e142ab3. `ClassCastException` is caught because queries like `SELECT 1` has a ValuesOperator that can't be casted to OpenSearchIndexScan which means it won't have `rawResponse`.